# Import necessary libraries

In [2]:
import pandas as pd
import numpy as np
from scipy import stats

## Data description 

α is given as 0.05 (95% Confidence)

Y is a Discrete data

X is a Discrete data

Since there are more than 2 variable we will perform Chi-Square test

## Chi-Square Test

In [3]:
df = pd.read_csv("Costomer+OrderForm.csv")

H0 == The defective % does not varies by centre  
H1 == The defective % does varies by centre

In [4]:
stats.chi2_contingency([df['Phillippines'].value_counts(), df['Indonesia'].value_counts(), df['Malta'].value_counts(), df['India'].value_counts()])

(3.8589606858203545,
 0.2771020991233144,
 3,
 array([[271.75,  28.25],
        [271.75,  28.25],
        [271.75,  28.25],
        [271.75,  28.25]]))

P value of Chi-Square test  == 0.277 > α

## Thus HO is accepted.  
## The defective % does not varies significantly by centres